components/TutorialEndCard.js

```diff
@@ -0,0 +1,46 @@
+import Image from "next/image";
+import React from "react";
+import styled from "styled-components";
+
+export default function TutorialEndCard() {
+  return (
+    <>
+      <CardContainer>
+        <Subline>100%</Subline>
+        <Image
+          src="/assets/tutorial-end-card_animation_300x300.gif"
+          alt="rocket animation"
+          width={300}
+          height={300}
+        ></Image>
+        <Subline>Congratulations!</Subline>
+        <StyledP>
+          Someone has shared knowledge with you.
+          <br />
```

Comment:
i heard that we shouldn't add breaks. just use 2 `p` tags

Reply:
Didn't get that memo! Thanks for letting me know. 😃